ksql-engine/src/test/java/io/confluent/ksql/embedded/KsqlContextTest.java

```diff
@@ -382,10 +384,14 @@ public void shouldSetProperty() {
     ksqlContext.sql("SQL;", ImmutableMap.of());
 
     // Then:
-    verify(ksqlEngine).execute(ConfiguredStatement.of(
-        PREPARED_STMT_0, ImmutableMap.of(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest"),
-        SOME_CONFIG
-    ));
+    verify(ksqlEngine, times(3)).execute(
```

Comment:
why 3 times now when previously only 1?

Reply:
Turns out that the issue was using this
```
verify(ksqlEngine).execute(
        ksqlEngine.getServiceContext(),
        ...
```
instead of
```
verify(ksqlEngine).execute(
        serviceContext
        ...
```

Mockito also verifies how many times the getServiceContext is called, or it is a bug or a conflict.